### PR TITLE
[10.0][fix] membership_extension: add security group to menu

### PR DESCRIPTION
**Pull request by luismontalba**
- Add access right to the main menu "Members" to employees group, avoiding restriction for only accountants when the association module is installed.

### CODE REVIEW

**Comment by pedrobaeza** [comment]
This is not security. It's changing a menu field, so it should go in /views/.

Any way, this is not what we have talked. I want a new security group for granting specifically that. And check all ACLs needed to that group. Imagine you only have that new group, and you need to access the same objects in that menu than any other role with that permission.

**Comment by chienandalu** [comment]
@pedrobaeza I've made the security group, and gave permissions so a user with just this group could access to the whole features of the  membership application.

File: membership_extension/__manifest__.py [hunk 1]
```diff
@@ -2,12 +2,13 @@
...
 # Copyright 2017 Pedro M. Baeza <pedro.baeza@tecnativa.com>
 # Copyright 2017 David Vidal <david.vidal@tecnativa.com>
+# Copyright 2017 Luis M. Ontalba <luis.martinez@tecnativa.com>
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
     "name": "Membership extension",
     "summary": "Improves user experience of membership addon",
-    "version": "10.0.1.1.3",
+    "version": "10.0.1.1.4",
```
Comment (pedrobaeza) [inline membership_extension/__manifest__.py]: Change version to 10.0.1.2.0

File: membership_extension/security/membership_security.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+<odoo>
+
+    <record model="ir.module.category" id="module_category_membership_management">
+        <field name="name">Membership</field>
+        <field name="description">Helps you manage operations for managing memberships.</field>
+        <field name="sequence">3</field>
+    </record>
+
+    <record id="group_membership_manager" model="res.groups">
+        <field name="name">Membership manager</field>
```
Comment (pedrobaeza) [inline membership_extension/security/membership_security.xml]: Call it only `Manager`, as the display name will be `Membership / Manager`

**Review by pedrobaeza — CHANGES_REQUESTED** [review]
Are you sure you don't need more permissions?

**Comment by chienandalu** [comment]
@pedrobaeza Changes done (added one ACL more) and tested that a user with just **Membership manager**, **Employee** and **Contact creation** permissions can do all member operations except billing (for wich the **Billing** permission would be granted)

**Review by pedrobaeza — APPROVED** [review]

**Review by rafaelbn — APPROVED** [review]
Functionally tested in runbot 👍

**Comment by rafaelbn** [comment]
Thanks @chienandalu

**Pull request merged**